sql/hive/src/test/scala/org/apache/spark/sql/hive/QueryPartitionSuite.scala

```diff
@@ -70,6 +71,45 @@ class QueryPartitionSuite extends QueryTest with SQLTestUtils with TestHiveSingl
     }
   }
 
+  test("Replace spark.sql.hive.verifyPartitionPath by spark.files.ignoreMissingFiles") {
+    withSQLConf((SQLConf.HIVE_VERIFY_PARTITION_PATH.key, "false")) {
```

Comment:
nit: we usually write `withSQLConf(SQLConf.HIVE_VERIFY_PARTITION_PATH.key -> "false")`